components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -4,14 +4,16 @@ import React from 'react';
 
 describe('LeagueEntries', () => {
   it(`renders 'default' state without a pick made`, () => {
-    render(<LeagueEntries entryName="Entry 1" linkUrl='' />);
+    render(<LeagueEntries entryName="Entry 1" linkUrl="" />);
 
     const leagueEntryContainerCard = screen.getByTestId(
       'league-entry-container-card',
     );
     const leagueEntryNumber = screen.getByTestId('league-entry-number');
     const entryStatus = screen.getByTestId('entry-status');
-    const leagueEntryPickButton = screen.getByTestId('league-entry-pick-button');
+    const leagueEntryPickButton = screen.getByTestId(
+      'league-entry-pick-button',
```

Comment:
Why is there a comma after  the identifier?

Reply:
When saving the code, VS auto populates the comma.

Reply:
Huh, weird.